pkg/liveness/liveness.go

```diff
@@ -58,13 +59,14 @@ func getLiveness(timeout time.Duration, csiConn *grpc.ClientConn) {
 }
 
 func recordLiveness(endpoint string, pollTime, timeout time.Duration) {
+	liveMetricsManager := metrics.NewCSIMetricsManager(conf.DriverName)
```

Comment:
conf should be the argument for this function or driver name should be the function argument.

Reply:
Done.